pipelines/keyboard/divvun-keyboard.ts

```diff
@@ -150,18 +303,39 @@ export function pipelineDivvunKeyboard() {
 export function pipelineDesktopKeyboard() {
   const pipeline: BuildkitePipeline = {
     steps: [
+      // TODO: 2025-09-04 re-enable this once windows bundling components are updated.
+      // This is turned off for now because it currently creates an installer that appears
+      // to work, but does not actually install a keyboard on Win10/Win11.
```

Comment:
Note Windows disabled for now for reasons above. Once Windows toolchain (kbdi, etc) are updated, this pipeline should Just Work.